backend/src/main/java/se/kth/iv1201/recruitmentapplicationgroup5/controller/ApiExceptionHandler.java

```diff
@@ -55,18 +55,19 @@
 			ConstraintViolationException e,
 			WebRequest req)
 	{
+		log.debug(e.getMessage());
 		return handleInternalError(e, req);
 	}
 	
 	@ExceptionHandler(value = Exception.class)
 	public ResponseEntity<Object> handleAllExceptions(Exception e, WebRequest req) {
+		log.debug(e.getMessage());
 		return handleInternalError(e, req);
 	}
 	
 	private ResponseEntity<Object> handleInternalError(Exception e, WebRequest req) {
-		//Log error here instead of System.out
-		System.out.println(e);
-		System.out.println(req);
+		log.debug(e.getMessage());
+		log.debug(e.getStackTrace().toString());
```

Comment:
This only prints the pointer to the array of stacks. I recommend changing this line to: 
log.debug(Arrays.toString(e.getStackTrace()).replaceAll(",", "\n"));

For other alternatives, solutions see https://stackoverflow.com/questions/1149703/how-can-i-convert-a-stack-trace-to-a-string